go.mod

```diff
@@ -1,12 +1,12 @@
 module gonum.org/v1/plot
 
 require (
-	github.com/ajstarks/svgo v0.0.0-20180226025133-644b8db467af
```

Comment:
How many of these changes were necessary for this fix?

Reply:
probably none of the version bumps are necessary.
I was just being sloppy: removing the `go.mod` and `go.sum` files (and then regenerate them with `GO111MODULE=on go mod init && go get ./...`) was easier than removing `draw2d` and tracking its deps...

I can sort this out if you prefer.

Reply:
done.